src/pages/index.js

```diff
@@ -127,11 +127,7 @@ const topics = [
       gender: 'male',
     },
     type: 'development',
-    title: (
-      <Fragment>
-        Отказоустойчивые решения&nbsp;PostgreSQL с автоматическим failover'ом
```

Comment:
`&nbsp;` and element usage instead of plaintext IMHO are not seeming reasonable there, moreover no-break whitespace makes string long and block mobile browser to truncate it (of course I'm not aware about hidden issues if there are existing).

Reply:
Это было сделано потому что на определенных разрешениях ломалось.
<img width="330" alt="image" src="https://user-images.githubusercontent.com/5055595/45492005-d3f57100-b773-11e8-9b51-1b9a85674bb3.png">

Reply:
Возможно тогда стоит подкрутить стили, мне пробел без переноса не дал этого понять. Если есть возможность, дай пожалуйста значения разрешения, где это ломается.

Reply:
Поправил стили, теперь нет перекрытия, с одним нюансом, плитки ровные в ряд, но сами ряды могут быть разной высоты из-за автоматической высоты, а также на экранах, где столбец только один. Было бы неплохо, если бы все плитки были одной высоты с подгонкой под самую большую (хотя неизвестно, насколько лучше, ведь тогда для коротких топиков будет много пустого места в плитке), но не вспомню сходу, можно ли это сделать без кода на чистом css. В любом случае, имхо это должно быть лучше, чем перекрытие текста или подгонка контента под код.
<img width="1151" alt="screen shot 2018-09-13 at 17 47 44" src="https://user-images.githubusercontent.com/3152460/45496288-bfb67180-b77d-11e8-945a-b0e9fd0b270f.png">

Reply:
@langovoi, посмотри пожалуйста по возможности.